packages/manager/apps/dedicated/client/app/ip/byoip/disclaimer/disclaimer.controller.js

```diff
@@ -0,0 +1,5 @@
+export default class IpByoipDisclaimer {
+  cancel() {
+    return this.goBack();
+  }
+}
```

Comment:
Useless controller, you can call `$ctrl.goBack()` directly in the template

Reply:
Useless except if maybe tracking planned.

Reply:
This is done keeping tracking in mind which we have to implement in the next sprint